### PR TITLE
update readme file describing how to change  desktop environments

### PR DESCRIPTION
**Pull request by itamarjp**
update readme file describing how to change  desktop environments

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -34,6 +34,43 @@ make
 then as root
 make install
 
+
+Customize Desktop Environment for xRDP Session
```
Comment (proski) [inline readme.txt]: Please use "XRDP" (preferred for the whole software package) or "xrdp" (preferred for the executable file), but not any other capitalization.

File: readme.txt [hunk 2]
```diff
@@ -34,6 +34,43 @@ make
...
+
+If you do not want to use the default desktop environment, you can customize it by creating a .Xclients file (X is capital!!!) in your home directory to launch the desktop environment you want and making it executable. In order to do this, open a terminal and run one of the following commands
+
+Gnome 3:
+echo "gnome-session" > ~/.Xclients
+chmod +x ~/.Xclients
+sudo systemctl restart xrdp.service
```
Comment (proski) [inline readme.txt]: Not all distros use systemd. Please move that line to the end, it's common for all recipes.

File: readme.txt [hunk 3]
```diff
@@ -34,6 +34,43 @@ make
...
 make install
 
+
+Customize Desktop Environment for xRDP Session
+
+If you do not want to use the default desktop environment, you can customize it by creating a .Xclients file (X is capital!!!) in your home directory to launch the desktop environment you want and making it executable. In order to do this, open a terminal and run one of the following commands
+
+Gnome 3:
+echo "gnome-session" > ~/.Xclients
+chmod +x ~/.Xclients
```
Comment (proski) [inline readme.txt]: It is really needed on any distro? I prefer not to have executable configuration files in my home directory.

Comment (moobyfr) [inline readme.txt]: I'm not sure `.Xclients` is correct, if a DM is used (`xdm`, `lightdm`...), the file to be used is `~/.xsession`. If the session is started from console, it has to be `.xinitrc`

**Comment by metalefty** [comment]
Readme has been converted to markdown. This can't be merged.

**Pull request closed without merging**